src/Service/SellerOnboardingService.php

```diff
@@ -117,13 +135,43 @@ protected function createStripeAccountFromShop(MiraklShop $shop): Account
                 'business_profile' => [
                     'name' => $rawShop['shop_name'] ?? null,
                     'url' => $rawShop['contact_informations']['web_site'] ?? null,
-                    'support_email' => $rawShop['contact_informations']['email'] ?? null,
-                    'support_phone' => $rawShop['contact_informations']['phone'] ?? null,
+                    'support_email' => $rawShop['contact_informations']['email'] ?? null
                 ],
             ];
+
+            if ($rawShop['contact_informations']['phone'] && $rawShop['contact_informations']['phone'] != '') {
+                $details['business_profile']['support_phone'] = $rawShop['contact_informations']['phone'];
+            }
+        }
+
+        $additionalMetaDataFields = $this->getAdditionalMetaDataFields($shop);
+
+        $metaData = array_merge($additionalMetaDataFields, [
+            'miraklShopId' => $shop->getId()
+        ]);
+
+        return $this->stripeClient->createStripeAccount($shop->getId(), $details, $metaData);
+    }
+
+    /**
+     * @param $shop
+     * @return array
+     */
+    private function getAdditionalMetaDataFields($shop)
+    {
+        $additionalMetaDataFields = [];
+        if ($this->stripeAccountMetadata !== null) {
+            $shopData = $shop->getShop();
+            $additionalMetaData = json_decode($this->stripeAccountMetadata);
+
+            foreach ($additionalMetaData as $fieldKey => $fieldValue) {
```

Comment:
Without the `STRIPE_ACCOUNT_METADATA` env var set, we're seeing a warning in the logs:
```
Warning: foreach() argument must be of type array|object, null given
/var/www/mirakl-stripe/src/Service/SellerOnboardingService.php:167
```